library/src/main/java/com/novoda/downloadmanager/LiteDownloadManagerCommands.java

```diff
@@ -29,9 +29,9 @@
 
     @Nullable
     @WorkerThread
-    DownloadFileStatus getDownloadStatusWithMatching(DownloadFileId downloadFileId);
+    DownloadFileStatus getDownloadStatusWithMatching(DownloadBatchId downloadBatchId, DownloadFileId downloadFileId);
```

Comment:
I just noticed the method says `DownloadStatus`, probably should be `FileDownloadStatus`.

Reply:
Would rather change it now because we'll need to update in client app anyway!